client/src/components/mapping/ConceptMap/index.tsx

```diff
@@ -9,6 +15,10 @@ import CodeSystemSelect from 'components/selects/codeSystemSelect';
 import StringSelect from 'components/selects/stringSelect';
 import { IReduxStore } from 'types';
 
+import UploadCodeSystem from 'components/uploads/uploadCodeSystem';
+import { validator } from 'components/uploads/validate';
```

Comment:
maybe we should put that in the `service` folder instead of `components` now that we use it in multiple components ?

Reply:
Currently, this folder is used for redux reducers and actions (maybe we can find another name)?
Do you want to get only the `components/uploads/validate.ts` file out of `components`?

Reply:
you're right it would not fit into the services folder. leave it here sinc this is used only in uploads...